### ISSUE TITLE
FreshCommand generates backtrace due to missing force option

### ISSUE THREAD

**Issue by luceos** [laraflock/dashboard#7]
When running `artisan dashboard:setup` the FreshCommand is executed which loads

``` php
        if (!$this->confirmToProceed()) {
            return;
        }
```

The `ConfirmableTrait` however expects a `--force` option on the command as seen on line 21:

``` php
    public function confirmToProceed($warning = 'Application In Production!', Closure $callback = null)
    {
        $shouldConfirm = $callback ?: $this->getDefaultConfirmCallback();

        if (call_user_func($shouldConfirm)) {
            if ($this->option('force')) {
                return true;
            }

            $this->comment(str_repeat('*', strlen($warning) + 12));
            $this->comment('*     '.$warning.'     *');
            $this->comment(str_repeat('*', strlen($warning) + 12));
            $this->output->writeln('');

            $confirmed = $this->confirm('Do you really wish to run this command? [y/N]');

            if (!$confirmed) {
                $this->comment('Command Cancelled!');

                return false;
            }
        }

        return true;
    }
```

Either add a force option to freshCommand or remove the confirm.

**Comment by iolson** [comment]
This will be pushed up in the next release.

**Comment by iolson** [comment]
Fixed in v3.1.0
